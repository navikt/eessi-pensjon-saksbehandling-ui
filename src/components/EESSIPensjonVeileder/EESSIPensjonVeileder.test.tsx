import { mount, ReactWrapper } from 'enzyme'
import React from 'react'
import EESSIPensjonVeileder from './EESSIPensjonVeileder'

describe('components/EESSIPensjonVeileder/EESSIPensjonVeileder', () => {
  let wrapper: ReactWrapper
  it('Render: match snapshot', () => {
    wrapper = mount(<EESSIPensjonVeileder />)
    expect(wrapper.isEmptyRender()).toBeFalsy()
    expect(wrapper).toMatchSnapshot()
  })

  it('Render: chooses veileder correctly', () => {
    wrapper = mount(<EESSIPensjonVeileder mood='smilende' />)
    expect(wrapper.exists('[data-test-id=\'c-eessipensjonveileder\']')).toBeTruthy()
    expect(wrapper.find('img').props().alt).toEqual('nav-smilende-veileder')
    wrapper.setProps({ mood: 'trist' })
    expect(wrapper.find('img').props().alt).toEqual('nav-trist-veileder')
  })
})
