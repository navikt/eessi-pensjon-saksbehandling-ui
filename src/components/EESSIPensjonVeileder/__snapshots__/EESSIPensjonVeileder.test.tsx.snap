// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/EESSIPensjonVeileder/EESSIPensjonVeileder Render: match snapshot 1`] = `
<EESSIPensjonVeileder>
  <div
    data-test-id="c-eessipensjonveileder"
  >
    <img
      alt="nav-smilende-veileder"
      height={130}
      src="navPensjonSmilendeOrangeVeileder.png"
      width={130}
    />
  </div>
</EESSIPensjonVeileder>
`;
